content/en/technical-overview/_index.md

```diff
@@ -2,11 +2,16 @@
 title: "Technical Overview of the CHT"
 linkTitle: "Technical Overview"
 weight: 3
-description: >
-  Technical overview and architecture of CHT components
 aliases:
    - /core
    - /core/overview/
 ---
 
-{{< subpages >}}
+{{< hextra/hero-subtitle >}}
```

Comment:
Following https://github.com/medic/cht-docs/pull/1886, we replaced `hextra/hero-subtitle` with the usual `description` header field to be able to re use the ols `relatedContent` shortcode. Please remove `hextra/hero-subtitle` and add back the `description`. Sorry for the confusion!